lib/spack/spack/cmd/create.py

```diff
@@ -668,6 +679,7 @@ def __call__(self, stage, url):
             (r'/Rakefile$',           'ruby'),
             (r'/setup\.rb$',          'ruby'),
             (r'/.*\.pro$',            'qmake'),
+            (r'/.*\.rockspec$',       'lua'),
```

Comment:
You can also update `lib/spack/spack/test/build_system_guess.py` to add tests for this.

Reply:
Thanks for that, added.